libraries/ts-command-line/src/providers/ScopedCommandLineAction.ts

```diff
@@ -48,6 +48,8 @@ class InternalScopedCommandLineParser extends CommandLineParser {
   }
 }
 
+export const SCOPING_PARAMETER_GROUP: unique symbol = Symbol('scoping');
```

Comment:
Why export and provide as a static property?

Reply:
Just easier to refer to its type this way.